app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -109,39 +116,40 @@ export default {
     properties() {
       return [
         {
-          title: "Name",
+          title: "Moniker",
           value: "small_moniker",
-          tooltip: "The unique moniker of this delegate.",
+          tooltip: "The validator's moniker",
           class: "name"
         },
         {
-          title: "% of Vote",
+          title: `% of ${this.bondingDenom}`,
           value: "percent_of_vote",
-          tooltip:
-            "The delegate controls this percentage of voting power on the network.",
+          tooltip: `Percentage of ${
+            this.bondingDenom
+          } the validator has on The Cosmos Hub`,
           class: "percent_of_vote"
         },
         {
-          title: "Total Votes",
+          title: `Total ${this.bondingDenom}`,
           value: "voting_power",
-          tooltip: `The delegate stakes this many ${
+          tooltip: `Total number of ${
             this.bondingDenom
-          }s on the network.`,
+          } the validator has on The Cosmos Hub`,
```

Comment:
Note: will eventually have to change "The Cosmos Hub" to a variable with the name of the zone